DashAI/front/package.json

```diff
@@ -31,7 +34,8 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "json-server": "json-server db.json --routes routes.json --port 3001"
```

Comment:
I haven't researched much on this, so please ignore this if it introduces too many complications. But I think it could be useful to have the option to use `.env.production` when starting with `yarn start`. 

Maybe adding something like this:
`"start:custom": "NODE_ENV=production craco start" `
 so that when doing `yarn start:custom`, `.env.production` can be used.